examples/seq2seq/seq2seq.py

```diff
@@ -136,7 +136,7 @@ def __init__(
         self.device = device
         self.max_length = max_length
 
-    def forward(self, trainer):
+    def __init__(self, trainer):
```

Comment:
Oh, it's `__init__`.

```suggestion
    def __call__(self, trainer):
```

Reply:
Sorry I somehow made this weird mistake... fixed.